AmplifyPlugins/Auth/Podfile.lock

```diff
@@ -10,18 +10,18 @@ PODS:
     - AWSCore (~> 2.21.0)
     - AWSMobileClient (~> 2.21.0)
     - AWSPluginsCore (= 1.5.2)
-  - AWSAuthCore (2.21.0):
-    - AWSCore (= 2.21.0)
-  - AWSCognitoIdentityProvider (2.21.0):
-    - AWSCognitoIdentityProviderASF (= 2.21.0)
-    - AWSCore (= 2.21.0)
-  - AWSCognitoIdentityProviderASF (2.21.0)
-  - AWSCore (2.21.0)
-  - AWSMobileClient (2.21.0):
-    - AWSAuthCore (= 2.21.0)
-    - AWSCognitoIdentityProvider (= 2.21.0)
-    - AWSCognitoIdentityProviderASF (= 2.21.0)
-    - AWSCore (= 2.21.0)
+  - AWSAuthCore (2.21.1):
+    - AWSCore (= 2.21.1)
+  - AWSCognitoIdentityProvider (2.21.1):
+    - AWSCognitoIdentityProviderASF (= 2.21.1)
+    - AWSCore (= 2.21.1)
+  - AWSCognitoIdentityProviderASF (2.21.1)
+  - AWSCore (2.21.1)
+  - AWSMobileClient (2.21.1):
+    - AWSAuthCore (= 2.21.1)
+    - AWSCognitoIdentityProvider (= 2.21.1)
+    - AWSCognitoIdentityProviderASF (= 2.21.1)
+    - AWSCore (= 2.21.1)
```

Comment:
Feels like this changes should be removed? This will be automatically bump when doing the next Amplify release right?

Reply:
Removed. But I think it doesnot matter keeping it as well.